lib/config/fileconf.go

```diff
@@ -947,6 +948,9 @@ type Auth struct {
 	// LoadAllCAs tells tsh to load the CAs for all clusters when trying
 	// to ssh into a node, instead of just the CA for the current cluster.
 	LoadAllCAs bool `yaml:"load_all_cas,omitempty"`
+
+	// Plugins configures the hosted plugins runtime
+	Plugins Plugins `yaml:"plugins,omitempty"`
```

Comment:
Not sure if this is best place to put it this, but my reasoning is: `plugins` is not really like the other standalone services - it does not have a `listen_addr`, and can only run inside Auth.

Reply:
If we want to be able to reuse the same configuration format for both standalone and in-auth modes, I'm not sure placing it under "auth" makes a lot of sense tbh. Why not a separate top-level section? Then we could reuse it for the new OpsGenie plugin too and have it be configured using same YAML fileconfig format rather than TOML.

And if we ever decide we want to make plugins into proper Teleport services (which I still think makes sense), it would be easier as well.